master/internal/webhooks/postgres_webhook.go

```diff
@@ -617,8 +747,22 @@ func generateSlackPayload(
 		}
 		c = "#DD5040"
 		mStatus = "Errored"
+	default:
+		status = fmt.Sprintf("The status of your experiment is %s", e.State)
+		if baseURLIsSet {
+			eURL = fmt.Sprintf("<%v/det/experiments/%v/overview | %v (#%v)>",
+				webUIBaseURL, e.ID, activeConfig.Name(), e.ID)
+		} else {
+			eURL = fmt.Sprintf("%v (#%v)", activeConfig.Name(), e.ID)
+		}
+		mStatus = string(e.State)
+	}
+
+	endTime := time.Now()
```

Comment:
Why we need to set endTime to time.Now()?

Reply:
I could be wrong but it seems like `endTime` is only used here to set the experiment duration in `expBlockFields`, so if it hasn't completed yet (`endTime == nil`), then we can get the duration from start time up until the present!

Reply:
Ah thanks! That calculates the experiment duration for a Slack message.

Reply:
Exactly, thanks @amandavialva01